src/dotnet/Db/Module/DbModule.cs

```diff
@@ -109,6 +165,51 @@ public class DbModule : HostModule<DbSettings>
         });
     }
 
+    private static string DumpConnections(DbConnection dbConnection)
```

Comment:
Same for this piece.

Reply:
DbModule is definitely not the right place for this.

Reply:
this code is reverted